src/js/engagement_view/src/components/graphDisplay/GraphDisplay.tsx

```diff
@@ -24,6 +24,7 @@ const defaultGraphDisplayState = (
     return {
         graphData: { nodes: [], links: [], index: {} },
         curLensName: lensName,
+        toggle: false,
```

Comment:
I'd just add a comment like `// toggle lets us explicitly trigger the useEffect by flipping the state`

Reply:
Removed toggle feature - unnecessary